README.md

```diff
@@ -1982,6 +1982,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Surbhi Pathak](https://github.com/surbhiahuja)
   - [Surya Tej Nimmakayala](https://github.com/CodeLapseLogger)
   - [Suryanshu Tomar](https://github.com/SuryanshuTomar)
+  - [Surya Teja](https://github.com/Surya-Teja-Sankuri)
```

Comment:
Hey Surya, I believe your position should be at line 1984 instead of 1985

Reply:
Ok I'll change. Thank you for helping me 🙂